### DISCUSSION TITLE
Different coloured bond notes or atom notes on a single image

### DISCUSSION THREAD

**Discussion by johncalpine** [rdkit/rdkit#6624] [Development]
In an rdkit generated image, Is it possible to display different bond notes in different colour?

**Comment by DavidACosgrove** [comment]
Hi,
This is also not possible.  Again, you can add different coloured notes
yourself using drawString() and setColour().  Working out where to place
the note is non-trivial.  If you are working in C++ you can use the
underlying functions for finding a good position for a note, but that is
not part of the supported public interface so not for the fainthearted.
Dave


On Fri, Aug 11, 2023 at 1:57 PM johncalpine ***@***.***>
wrote:

> In an rdkit generated image, Is it possible to display different bond
> notes in different colour?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rdkit/rdkit/discussions/6624>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACGF2FTGSSPFRW7S6HIKHPLXUYT2FANCNFSM6AAAAAA3M4L7FE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
David Cosgrove
Freelance computational chemistry and chemoinformatics developer
http://cozchemix.co.uk

**Comment by johncalpine** [comment]
Thanks Dave
